kvdb-rocksdb/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog].
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
+- Removed `owning_ref` from dependencies :tada:. [662](https://github.com/paritytech/parity-common/pull/662)
+### Breaking
+- Update `kvdb` to 0.12. [662](https://github.com/paritytech/parity-common/pull/662)
+  - `add_column` and `remove_last_column` now require `&mut self`
```

Comment:
```suggestion
  - `add_column` and `remove_last_column` now require `&mut self`
  - remove `ReadGuardedIterator`
```
Did not find it used anywhere though

Reply:
It was not exposed publicly. The `iter` module is private.